ctapipe/reco/reco_algorithms.py

```diff
@@ -0,0 +1,23 @@
+from ctapipe.core import Component
+from ctapipe.io.containers import RecoShowerGeom
+
+
+class RecoShowerGeomAlgorithm(Component):
+    """This is the base class from which all direction reconstruction algorithms should inherit from"""
+
+    def __init__(self, model = None):
```

Comment:
don't override `__init__`  here, the current way Components work will break if you do that.  Probably we need to think about refactoring them though.     

What is the `model` parameter used for? Can it be removed or made a trait property of the class?